### PR TITLE
Added variables for stock location

### PR DESCRIPTION
**Pull request by SergeoLacruz**

### CODE REVIEW

File: docs/report/context_variables.md [hunk 1]
```diff
@@ -150,8 +150,16 @@ Each part object has access to a lot of context variables about the part. The fo
...
 |----------|-------------|
-| barcode | Brief payload data (e.g. for labels) |
-| item_count | Simply returns the number of stock items in this location. | 
+| barcode | Brief payload data (e.g. for labels). Example: {"stocklocation": 826} where 826 is the primary key|
+| description | The description of the location  |
+| icon | The name of the icon if set, e.g. fas fa-warehouse |
+| item_count | Simply returns the number of stock items in this location | 
+| name | The name of the location. This is only the name of this location, not the path | 
+| owner | The owner of the location if it has one. The owner can actually only be assigned in the admin interface | 
+| parent | The parent location one level up. Returns None if it is already the top most one |
```
Comment (matmair) [inline docs/report/context_variables.md:159]: Question: Isn't this a bit redundant? The parent location is already one level up

File: docs/report/context_variables.md [hunk 2]
```diff
@@ -150,8 +150,16 @@ Each part object has access to a lot of context variables about the part. The fo
...
+| barcode | Brief payload data (e.g. for labels). Example: {"stocklocation": 826} where 826 is the primary key|
+| description | The description of the location  |
+| icon | The name of the icon if set, e.g. fas fa-warehouse |
+| item_count | Simply returns the number of stock items in this location | 
+| name | The name of the location. This is only the name of this location, not the path | 
+| owner | The owner of the location if it has one. The owner can actually only be assigned in the admin interface | 
+| parent | The parent location one level up. Returns None if it is already the top most one | 
+| path | A query set of locations that contains the hierarchy starting from the top most parent | 
+| path_string | A string that contains all all names of the path separated by slashes e.g. A/B/C | 
+| structural | A boolean the tells if the location is structural |
```
Comment (matmair) [inline docs/report/context_variables.md:162]: Suggestion
```suggestion
| structural | True if the location is structural | 
```

**Review by matmair — APPROVED** [review]
Thank you for your contribution! I have some suggestions regarding the phrasing.

**Comment by SergeoLacruz** [comment]
Done :-)

**Review by matmair — APPROVED** [review]

**Comment by matmair** [comment]
Thanks again, we will merge once the maintainer is back online in a few days.

**Comment by SchrodingersGat** [comment]
@SergeoLacruz many thanks for the contributions :)

**Pull request merged**